src/functions.jl

```diff
@@ -0,0 +1,8 @@
+# This file is a part of InverseFunctions.jl, licensed under the MIT License (MIT).
+
+"""
+    square(x)
+
+Inverse of `sqrt(x)`.
```

Comment:
Maybe add we only consider the non-negative real line for `sqrt` and `square`?
```suggestion
Inverse of `sqrt(x)` for non-negative `x`.
```